lib/resources/dstu2/immunization.yaml

```diff
@@ -0,0 +1,77 @@
+---
+name: Immunization
+field_name_base_url: http://hl7.org/fhir/immunization-definitions.html#Immunization
+fields:
+
+- name: vaccineCode
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: The vaccine that was administered or was to be administered.
+  example: |
+            {"coding": [
+              {"system": "http://loinc.org",
+               "code": "34840-9"}]}
+            {"coding": [
+              {"system": "http://www2a.cdc.gov/vaccines/iis/iisstandards/vaccines.asp?rpt=cvx",
+               "code": "43"}],
+             "text": "Engerix-B (obsolete)"}
+  binding:
+    description: Identifies the vaccine substance administered - CVX codes.
```

Comment:
Probably don't need the '- CVX codes.' since that it indicated on the next line in the table.